frontend/packages/dev-console/src/utils/shared-submit-utils.ts

```diff
@@ -78,6 +78,13 @@ export const createService = (
     const port = { containerPort: _.toInteger(unknownTargetPort), protocol: 'TCP' };
     ports = [...ports, port];
```

Comment:
I think it would be better if we filter out the default port while adding the target port itself:
```suggestion
    ports = [...ports.filter((port) => port.containerPort !== defaultUnknownPort), port];
```